tests/integration/test_dynamic_tags.py

```diff
@@ -0,0 +1,56 @@
+"""
+Copyright (c) 2021 http://reportportal.io .
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License
+"""
+
+import uuid
+
+from delayed_assert import assert_expectations, expect
+from six.moves import mock
+
+from tests import REPORT_PORTAL_SERVICE
+from tests.helpers import utils
+
+TEST_CASES = ['Test tag set', 'Test no tag', 'Test set multiple tags']
+TEST_CASE_UUIDS = [str(uuid.uuid5(uuid.NAMESPACE_OID, i)) for i in TEST_CASES]
+
+
+def generate_id(*args, **kwargs):
+    return str(uuid.uuid5(uuid.NAMESPACE_OID, str(kwargs['name'])))
+
+
+@mock.patch(REPORT_PORTAL_SERVICE)
+def test_launch_log(mock_client_init):
+    mock_client = mock_client_init.return_value
+    mock_client.start_test_item.side_effect = generate_id
+
+    result = utils.run_robot_tests(['examples/dynamic_tags.robot'])
+    assert result == 0  # the test successfully passed
+
+    start_tests = [call for call in mock_client.start_test_item.call_args_list
```

Comment:
```suggestion
    start_tests = [
        call for call in mock_client.start_test_item.call_args_list if
        call[1]['item_type'] == 'STEP' and call[1].get('has_stats', True)]
```

Reply:
Mmmm... Have not noticed any changes...

Reply:
@iivanou OK, changed these lines right exactly as you posted them.